webapp/javascript/ui/Sidebar.scss

```diff
@@ -130,3 +130,11 @@ $sidebar-collapsed-width: 54px !default;
   .pro-inner-item {
   padding-left: 42px;
 }
+
```

Comment:
the reason why that happend is that SideBar imported styles use rgba(var(), 0.2), but rgba() doesnt work. if we use only var() it works fine. If we use rbga with no variable it doesnt work as well. maybe you have any ideas what can we wrong ?